4.0/init.sh

```diff
@@ -52,12 +69,30 @@ chown -R nominatim:nominatim ${PROJECT_DIR}
 
 cd ${PROJECT_DIR}
 sudo -E -u nominatim nominatim import --osm-file $OSMFILE --threads $THREADS
+
+if [ -f tiger2021-nominatim-preprocessed.csv.tar.gz ]; then
+  echo "Importing Tiger address data"
+  sudo -u nominatim nominatim add-data --tiger-data tiger2021-nominatim-preprocessed.csv.tar.gz
+  echo NOMINATIM_USE_US_TIGER_DATA=yes >> ${PROJECT_DIR}/.env
```

Comment:
We manipulate the config file in config.sh. Can you move the line 76 and 77 there, please?